installation/aws/VmClarity.cfn

```diff
@@ -152,6 +152,18 @@ Resources:
             apt:
               docker.io: []
           files:
+            "/etc/vmclarity/network_config.sh":
```

Comment:
what is this for?

Reply:
this script configures the docker network for the VMClarity services that allows the containers to talk and discover each other via DNS names. I created a script because we need to be able to handle the case during stack upgrade that the network already exists which isn't easy to do in just cloud-init commands.